central/detection/lifecycle/manager_impl.go

```diff
@@ -170,7 +204,13 @@ func (m *managerImpl) flushIndicatorQueue() {
 
 	// Group the processes into particular baseline segments
 	baselineMap := make(map[processBaselineKey][]*storage.ProcessIndicator)
```

Comment:
None of this baseline stuff should be needed anymore right? besides the:

m.queuedIndicators[indicator.GetId()] = indicator at the bottom

Reply:
I think we still have to do that stuff.  We will come through here as we get new indicators after we have forged a baseline and while poorly named, this is basically just building the stuff to pass to checkAndUpdateBaseline which will flag the risks for indicators that are not in the baseline.  Essentially it is building a map of the baselinekey to indicator so that it can retrieve the baseline and process the indicators against it in checkAndUpdateBaseline.  if I'm reading it correctly